src/lib.rs

```diff
@@ -54,7 +54,7 @@
 //! to the network through any node, and exchange public keys with it. That node becomes a
 //! bootstrap node for the client, and messages to and from the client will be routed over it.
 //!
-//! ```no_run
+//! ```ignore
```

Comment:
It can in non-testing mode, but in testing mode, the constructor for `Client` no longer takes an event-sender argument. We could conditionally include different examples here using `#cfg[...]` and `#[doc = ...]` but I was hesitant to spend time doing that unless you think it's worthwhile.